frontend/src/metabase/css/core/colors.module.css

```diff
@@ -29,6 +29,26 @@
   --mb-color-filter: #7172ad;
   --mb-color-focus: #cbe2f7;
   --mb-color-warning: #f9cf48;
+  --mb-color-border-alpha-30: color-mix(
```

Comment:
ignore, this will get yanked out into another PR... just needed this to make UI not broken anymore